src/Main/Global.h

```diff
@@ -13,75 +13,69 @@
 
 namespace carta {
 
-class GlobalValues {
+class Global {
```

Comment:
Please refer to my previous comment -- there is no need for this wrapper class at all. This functionality for creating and retrieving a singleton instance should be directly in the `ProgramSettings` class. The getters are unnecessary (the properties on the settings object can be accessed directly, as before).